.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ executors:
   ruby-latest:
     resource_class: small
     docker:
-      - image: circleci/ruby:latest
+      - image: circleci/ruby:2.6.6
```

Comment:
Can we switch this over to `cimg/ruby:2.7`? The circleci/ruby images are no longer maintained and most of our apps are now using 2.7.

Reply:
Done